client/me/controller.js

```diff
@@ -19,7 +19,7 @@ import purchasesController from './purchases/controller';
 import userFactory from 'lib/user';
 import userSettings from 'lib/user-settings';
 import titleActions from 'lib/screen-title/actions';
-import { Provider } from 'react-redux';
+import ReactHelpers from 'lib/react-helpers';
```

Comment:
`React` library introduces a lot of noise to our coding naming conventions:
https://github.com/Automattic/wp-calypso/blob/master/docs/coding-guidelines/javascript.md#naming-conventions.
I guess we should have here:
```js
import reactHelpers from 'lib/react-helpers';
// or even
import { renderWithReduxStore } from 'lib/react-helpers';
```